cypress/support/commands.js

```diff
@@ -42,13 +42,17 @@ Cypress.Commands.add('getTenureFixture', () => {
 })
 
 Cypress.Commands.add('getPatchFixture', () => {
-    cy.readFile('cypress/fixtures/PatchesAndAreas.json')
+    cy.readFile('cypress/fixtures/patches.json')[0]
```

Comment:
Should the JSON file be a list? also, I think the naming should be as it was i.e. `patchesAndAreas`

Reply:
For this, there actually isn't a PatchesAndAreas fixture anywhere, and other tests reference the "patches" fixture. We could rename all of them but when I tried it actually gets tricky because it's also .gitignored :laughing: 

So I'm not sure this is worth changing